package.json

```diff
@@ -67,6 +67,7 @@
     "eslint-plugin-jsx-a11y": "^6.8.0",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
+    "msw": "^2.3.0",
```

Comment:
q: why was msw added?

Reply:
Ah. That was part of my testing and I forgot to remove it. 

Will uninstall on my next push.